### PR TITLE
Nodejs week2

### PR DESCRIPTION
**Pull request by Atefeh-Behroozi**
Finished Warmup.

### CODE REVIEW

File: nodejs/week2/app.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      const values = Object.values(doc);
+      return values.some((value) => {
+        if (typeof value === "string" && value.includes(query)) {
+          return true;
+        }
+      });
+    });
+      res.json(filteredDocuments);
+  }
+});
```
Comment (raihanmehran) [inline nodejs/week2/app.js:27]: Nice Job 👍

File: nodejs/week2/app.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+app.get("/documents/:id", (req, res) => {
+  const id = parseInt(req.params.id);
+  const document = documents.find((doc) => doc.id === id);
```
Comment (raihanmehran) [inline nodejs/week2/app.js:30]: Verify the `id` field whether it is a number or not

**Review by raihanmehran — COMMENTED** [review]
In general, nice solution

File: nodejs/week2/app.js (hunk 1 above)
Comment (Atefeh-Behroozi) [reply nodejs/week2/app.js:27]: Thank you :)

**Comment by Atefeh-Behroozi** [comment]
Thank you for the feedback. I've fix it.

**Pull request merged**